pdfminer/arcfour.py

```diff
@@ -1,22 +1,19 @@
-
-
 """ Python implementation of Arcfour encryption algorithm.
 See https://en.wikipedia.org/wiki/RC4
 This code is in the public domain.
 
 """
 
-import six # Python 2+3 compatibility
 ##  Arcfour
 ##
-class Arcfour(object):
```

Comment:
I understand that removing explicit inheritence of `object` is possible since this is the default in Python3. However, I prefer that the inheritance stays explicit cause it's more consistent.